### PR TITLE
FPO-271-Header links

### PR DESCRIPTION
**Pull request by Lee365**
### Jira link
https://transformuk.atlassian.net/browse/FPO-271

### What?

I have added/removed/altered:

- [x] Added
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

-

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: src/config/main-navigation-options.ts [hunk 1]
```diff
@@ -4,6 +4,18 @@ export default function (req: Request, res: Response, next: NextFunction): void
   const mainNavigation: any = []
 
   if (req.oidc?.isAuthenticated()) {
+    mainNavigation.push({
+      href: '/dashboard',
+      text: 'Dashboard'
+    })
+    mainNavigation.push({
+      href: '/profile',
```
Comment (mkayad) [inline src/config/main-navigation-options.ts:12]: do you have a profile page to link to?

File: src/config/main-navigation-options.ts [hunk 2]
```diff
@@ -4,6 +4,18 @@ export default function (req: Request, res: Response, next: NextFunction): void
...
+      text: 'Update Profile'
+    })
+    mainNavigation.push({
+      href: '/team',
```
Comment (mkayad) [inline src/config/main-navigation-options.ts:16]: do you have a team page to link to?

Comment (Lee365) [reply src/config/main-navigation-options.ts:16]: This page is not ready yet and will be linked later.

File: src/config/main-navigation-options.ts (hunk 1 above)
Comment (Lee365) [reply src/config/main-navigation-options.ts:12]: This page is not ready yet and will be linked later.

Comment (willfish) [inline src/config/main-navigation-options.ts:12]: So I think we'll want to get these from the OIDC user profile attached to the session, here: https://github.com/trade-tariff/trade-tariff-dev-hub-frontend/blob/main/src/config/scpAuth.ts

```typescript
    const userProfile = req.appSession?.userProfile ?? null
```

Comment (willfish) [reply src/config/main-navigation-options.ts:12]: If you deploy a version of this with a console.log of the userProfile you should be able to work out which links are relevant in ECS.

I added a user for your email address, here: https://github.com/trade-tariff/trade-tariff-platform-terraform-aws-accounts/pull/32

So you should be able to follow the forgotten password flow for aws, here: https://d-9c677042e2.awsapps.com/start/#/

And then check ECS logs for the frontend application via the ECS service for the hub-frontend

**Review by willfish — APPROVED** [review]

**Pull request merged**